### PR TITLE
openshift/hypershift: fix tag naming

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ images:
       - registry.svc.ci.openshift.org/openshift/release:golang-1.15
   to: hypershift
 promotion:
-  name: "4.7"
+  name: latest
```
Comment (hongkailiu) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:25]:
```suggestion
  tag: latest
```

**Review by hongkailiu — APPROVED** [review]
/hold cancel

**Comment by ironcladlou** [comment]
Had to re-generate, please re-tag

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**